superset/connectors/sqla/models.py

```diff
@@ -251,7 +251,6 @@ class SqlaTable(Model, BaseDatasource):
     owner_class = security_manager.user_model
 
     __tablename__ = 'tables'
-    __table_args__ = (UniqueConstraint('database_id', 'table_name'),)
```

Comment:
Removed any table constraints as suggested by @mistercrunch

Reply:
@agrawaldevesh I'm not sure why we should be removing this uniqueness constraint. I've actually proposed having a more restrictive uniqueness constraint per https://github.com/apache/incubator-superset/pull/5449.